src/content/docs/mobile-monitoring/new-relic-mobile/maintenance/configure-network-request-settings.mdx

```diff
@@ -213,4 +213,13 @@ Now that you've seen some example of how this works, let's look at some examples
   </tbody>
 </table>
 
-Remember, a wildcard will only affect a whole segment. Since the `*` can only be used for a full segment, the `*` here will be taken literally and not as a wildcard.
\ No newline at end of file
+Remember, a wildcard will only affect a whole segment. Since the `*` can only be used for a full segment, the `*` here will be taken literally and not as a wildcard.
+
+## Query network request settings via NRQL
+
+You can use NRQL to query network request settings to do things like (insert use cases). Try out [these sample NRQL queries](/docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/nrql-query-examples-mobile-monitoring#network-request-settings).
```

Comment:
@a-sassman I think you need to add use cases above?

Reply:
Ah yes, I meant to remove this. Thanks for the catch.